modules/uhsg_rest/src/Plugin/rest/resource/DegreeProgrammeResource.php

```diff
@@ -123,4 +128,33 @@ private function getNameTranslations(TermInterface $term) {
     return $nameTranslations;
   }
 
+  /**
+   * @param \Drupal\taxonomy\TermInterface $term
+   * @param string content-type
+   * @return int
+   */
+  private function getNodeCount(TermInterface $term, $content_type) {
+    $contentCount = null;
+
+    $languages = $this->languageManager->getLanguages();
+    $languageCodes = array_map(function ($language) {
+      /** @var $language LanguageInterface */
+      return $language->getId();
+    }, $languages);
+
+    foreach ($languageCodes as $languageCode) {
+      $nodes[$languageCode] = \Drupal::entityTypeManager()->getStorage('node')->getQuery()
```

Comment:
I wonder if there is a significant performance cost in this query, but if there is, New Relic should tell us.
Code looks fine anyway.

Reply:
There shouldn't be, it's quite an easy query for the database. We only get nid's, not full nodes. And as you said, new relic will show this.